src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,143 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public static final int OK_YEARS_OLD = 18;
+    public static final int NOT_OK_YEARS_OLD = 17;
+    public static final int NOT_OK_MINUS_YEARS_OLD = -17;
+    public static final int NOT_OK_ZERO_YEARS_OLD = 0;
+    public static final String OK_PASSWORD = "password";
+    public static final String OK_LOGIN = "login12345";
+
+    private static RegistrationService registration = new RegistrationServiceImpl();
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_NullLogin_NotOk() {
+        User user1 = new User();
+        user1.setPassword(OK_PASSWORD);
+        user1.setAge(OK_YEARS_OLD);
+
+        assertThrows(RegisterException.class, () -> registration.register(user1),
+                "Login cant be null");
+    }
+
+    @Test
+    void register_NullPassword_NotOk() {
+        User user1 = new User();
+        user1.setLogin(OK_LOGIN);
+        user1.setAge(OK_YEARS_OLD);
+
+        assertThrows(RegisterException.class, () -> registration.register(user1),
+                "Password cant be null");
+    }
+
+    @Test
+    void register_BelowOrZeroAge_NotOk() {
+        User user1 = new User();
+        user1.setLogin("login123");
```

Comment:
When you test age here, it's better to add the constant for OK_LOGIN you created - it makes code more readable and inform that everything except age is okay.

You still hardcode a lot of values in diffrent tests, please create/use existing constants.